lib/engines/content_block_manager/app/components/content_block_manager/content_block_edition/details/fields/govspeak_enabled_textarea_component.rb

```diff
@@ -15,10 +15,24 @@ def initialize(content_block_edition:, value:, nested_object_key:, field:, subsc
 
   attr_reader :content_block_edition, :nested_object_key, :field, :subschema, :errors
 
+  def govspeak_enabled?
+    subschema.govspeak_enabled?(nested_object_key: nested_object_key, field_name: field.name)
+  end
+
   def id_attribute
     "#{PARENT_CLASS}_details_#{subschema_block_type}_#{nested_object_key}_#{field.name}"
   end
 
+  def aria_described_by
+    "#{id_attribute}-hint"
+  end
+
+  def hint_text
+    return nil unless govspeak_enabled?
+
+    "Markdown supported"
```

Comment:
Just wondering if we should refer to "Govspeak" rather than "Markdown". Is "Govspeak" better understood, or do content designers see the terms as interchangable?

Reply:
thanks for this, I've now made it "Govspeak" all the way down!